filament/backend/include/backend/Handle.h

```diff
@@ -106,8 +107,16 @@ struct Handle : public HandleBase {
     Handle(Handle const& rhs) noexcept = default;
     Handle(Handle&& rhs) noexcept = default;
 
-    Handle& operator=(Handle const& rhs) noexcept = default;
-    Handle& operator=(Handle&& rhs) noexcept = default;
+    // Explicitly redefine copy/move assignment operators rather than just using default here.
+    // Because it doesn't make a call to the parent's method automatically in certain compilers.
```

Comment:
That's extremely strange. What compiler(s) are affected? What's the effect? Is `HandleId object` not copied correctly?

Reply:
This issue occurs when `std::move(fences.begin() + 1, fences.end(), fences.begin());` is called (FrameSkipper.cpp:69) for specific NDK versions (<= 25.1.8937393, works fine from 26). Let me update the descriptions and comments.